src/components/accessories/XAxis/XAxis.jsx

```diff
@@ -20,10 +20,13 @@ export default class XAxis extends React.Component {
 
     xScale.rangeRound([0, containerWidth]);
 
-    $xAxis.call(
-      d3.axisBottom(xScale)
-        .tickPadding(10)
-    );
+    let axis = d3.axisBottom(xScale).ticks(d3.timeHour.every(3));
```

Comment:
ticks should be configurable, not hardcoded to timeHour.

Arguably, configuration should be standardized on [`tickArguments`](https://github.com/d3/d3-axis/blob/master/README.md#axis_tickArguments), which is a single interface that can configure both `ticks` and `tickFormat`.

For a shorter-term solution, though, it should just match YAxis.